config_project_creation.go

```diff
@@ -0,0 +1,66 @@
+package evergreen
+
+import (
+	"github.com/mongodb/anser/bsonutil"
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+type OwnerRepo struct {
+	Owner string `bson:"owner" json:"owner" yaml:"owner"`
+	Repo  string `bson:"repo" json:"repo" yaml:"repo"`
+}
+
+type ProjectCreationConfig struct {
+
+	// Total number of projects that evergreen is willing to support
+	TotalProjectLimit int `bson:"total_project_limit" json:"total_project_limit" yaml:"total_project_limit"`
```

Comment:
tiny nit: Evergreen (and 22)
Also variable comments should start with the name of the thing: TotalProjectLimit is the total number of.....

Reply:
updated